<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>691</width>
    <height>585</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>自动驾驶公交车调度系统 AutoBusManager 1.0</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="logo">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>30</y>
      <width>71</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:img/icon.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>631</width>
      <height>441</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>等线</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>开始</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_3">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>250</y>
        <width>221</width>
        <height>151</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>等线</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox{

border-width:1px;

border-style:solid;

border-radius: 10px;

border-color:gray;

margin-top:0.5ex;

}

QGroupBox::title{

subcontrol-origin:margin;

subcontrol-position:top left;

left:10px;

margin-left:0px;

padding:0 1px;

}</string>
      </property>
      <property name="title">
       <string>发出指令</string>
      </property>
      <widget class="QComboBox" name="type">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>40</y>
         <width>131</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>clockwise</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>counterclockwise</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>target</string>
        </property>
       </item>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>40</y>
         <width>31</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>等线</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>类型</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>70</y>
         <width>31</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>等线</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>站号</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="number">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>70</y>
         <width>131</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="submit">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>110</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>等线</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>提交指令</string>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="board">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>571</width>
        <height>191</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>等线</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox{

border-width:1px;

border-style:solid;

border-radius: 10px;

border-color:gray;

margin-top:0.5ex;

}

QGroupBox::title{

subcontrol-origin:margin;

subcontrol-position:top left;

left:10px;

margin-left:0px;

padding:0 1px;

}</string>
      </property>
      <property name="title">
       <string>面板</string>
      </property>
      <widget class="QLabel" name="track">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>60</y>
         <width>421</width>
         <height>81</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">image: url(:/img/track.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="bus">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>60</y>
         <width>16</width>
         <height>16</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-image: url(:/img/bus.png);</string>
       </property>
       <property name="text">
        <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
      <widget class="QPushButton" name="test">
       <property name="geometry">
        <rect>
         <x>230</x>
         <y>140</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>test</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>240</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>等线</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>状态栏</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>250</y>
        <width>91</width>
        <height>151</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>等线</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox{

border-width:1px;

border-style:solid;

border-radius: 10px;

border-color:gray;

margin-top:0.5ex;

}

QGroupBox::title{

subcontrol-origin:margin;

subcontrol-position:top left;

left:10px;

margin-left:0px;

padding:0 1px;

}</string>
      </property>
      <property name="title">
       <string>控制</string>
      </property>
      <widget class="QPushButton" name="start">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>31</width>
         <height>31</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>自动运行</string>
       </property>
       <property name="styleSheet">
        <string notr="true">background-image: url(:img/start.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="pause">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>40</y>
         <width>31</width>
         <height>31</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>暂停自动运行</string>
       </property>
       <property name="styleSheet">
        <string notr="true">background-image: url(:img/pause.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="step">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>31</width>
         <height>31</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>单步运行</string>
       </property>
       <property name="styleSheet">
        <string notr="true">background-image: url(:img/step.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="stop">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>80</y>
         <width>31</width>
         <height>31</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>结束运行</string>
       </property>
       <property name="styleSheet">
        <string notr="true">background-image: url(:img/stop.png);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </widget>
     <widget class="QTableWidget" name="status">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>260</y>
        <width>221</width>
        <height>141</height>
       </rect>
      </property>
      <property name="editTriggers">
       <set>QAbstractItemView::NoEditTriggers</set>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::NoSelection</enum>
      </property>
      <attribute name="horizontalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>100</number>
      </attribute>
      <attribute name="horizontalHeaderHighlightSections">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>25</number>
      </attribute>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderCascadingSectionResizes">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderDefaultSectionSize">
       <number>27</number>
      </attribute>
      <attribute name="verticalHeaderHighlightSections">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderShowSortIndicator" stdset="0">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <row>
       <property name="text">
        <string>TIME</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Position</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Clockwise</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>CounterClockwise</string>
       </property>
      </row>
      <row>
       <property name="text">
        <string>Target</string>
       </property>
      </row>
      <column>
       <property name="text">
        <string>值</string>
       </property>
       <property name="textAlignment">
        <set>AlignCenter</set>
       </property>
      </column>
      <item row="0" column="0">
       <property name="text">
        <string/>
       </property>
       <property name="flags">
        <set>NoItemFlags</set>
       </property>
      </item>
      <item row="1" column="0">
       <property name="text">
        <string/>
       </property>
       <property name="flags">
        <set>NoItemFlags</set>
       </property>
      </item>
      <item row="2" column="0">
       <property name="text">
        <string/>
       </property>
       <property name="flags">
        <set>NoItemFlags</set>
       </property>
      </item>
      <item row="3" column="0">
       <property name="text">
        <string/>
       </property>
       <property name="flags">
        <set>NoItemFlags</set>
       </property>
      </item>
      <item row="4" column="0">
       <property name="text">
        <string/>
       </property>
       <property name="flags">
        <set>NoItemFlags</set>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>210</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>clockwise</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>210</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>counterclockwise</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>550</x>
        <y>210</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>target</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>210</y>
        <width>41</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string> 图例：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>210</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/img/clo.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>210</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/img/cnt.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>210</y>
        <width>21</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/img/tar.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <zorder>board</zorder>
     <zorder>groupBox_3</zorder>
     <zorder>label_4</zorder>
     <zorder>groupBox_5</zorder>
     <zorder>status</zorder>
     <zorder>label_6</zorder>
     <zorder>label_7</zorder>
     <zorder>label_8</zorder>
     <zorder>label_9</zorder>
     <zorder>label_10</zorder>
     <zorder>label_11</zorder>
     <zorder>label_12</zorder>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>设置</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>10</y>
        <width>351</width>
        <height>231</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>等线</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox{

border-width:1px;

border-style:solid;

border-radius: 10px;

border-color:gray;

margin-top:0.5ex;

}

QGroupBox::title{

subcontrol-origin:margin;

subcontrol-position:top left;

left:10px;

margin-left:0px;

padding:0 1px;

}</string>
      </property>
      <property name="title">
       <string>配置参数</string>
      </property>
      <widget class="QLabel" name="TS_label">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>70</y>
         <width>181</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>等线</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string> 车站数量（TOTAL_STATION）</string>
       </property>
      </widget>
      <widget class="QLabel" name="D_lable">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>100</y>
         <width>151</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>等线</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string> 车站间距（DISTANCE）</string>
       </property>
      </widget>
      <widget class="QLabel" name="S_lable">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>40</y>
         <width>141</width>
         <height>16</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>等线</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string> 调度策略（STRATEGY）</string>
       </property>
      </widget>
      <widget class="QPushButton" name="save">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>140</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>等线</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>保存并初始化</string>
       </property>
      </widget>
      <widget class="QComboBox" name="S_edit">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>40</y>
         <width>91</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>FCFS</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SSTF</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>SCAN</string>
        </property>
       </item>
      </widget>
      <widget class="QPushButton" name="browse">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>180</y>
         <width>111</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>等线</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>浏览配置文件..</string>
       </property>
      </widget>
      <widget class="QLabel" name="path">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>120</y>
         <width>261</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QSpinBox" name="TS_edit">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>70</y>
         <width>91</width>
         <height>22</height>
        </rect>
       </property>
       <property name="minimum">
        <number>2</number>
       </property>
       <property name="maximum">
        <number>10</number>
       </property>
       <property name="value">
        <number>5</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="D_edit">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>100</y>
         <width>91</width>
         <height>22</height>
        </rect>
       </property>
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>10</number>
       </property>
       <property name="value">
        <number>2</number>
       </property>
      </widget>
     </widget>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>260</y>
        <width>351</width>
        <height>141</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>等线</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QGroupBox{

border-width:1px;

border-style:solid;

border-radius: 10px;

border-color:gray;

margin-top:0.5ex;

}

QGroupBox::title{

subcontrol-origin:margin;

subcontrol-position:top left;

left:10px;

margin-left:0px;

padding:0 1px;

}</string>
      </property>
      <property name="title">
       <string>动画设置</string>
      </property>
      <widget class="QLabel" name="Sp_label">
       <property name="geometry">
        <rect>
         <x>40</x>
         <y>60</y>
         <width>161</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>运行速度（每clock耗时）</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="sp_edit">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>60</y>
         <width>91</width>
         <height>22</height>
        </rect>
       </property>
       <property name="decimals">
        <number>1</number>
       </property>
       <property name="minimum">
        <double>0.100000000000000</double>
       </property>
       <property name="maximum">
        <double>10.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>0.100000000000000</double>
       </property>
       <property name="value">
        <double>1.000000000000000</double>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>关于</string>
     </attribute>
     <widget class="QLabel" name="info">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>90</y>
        <width>401</width>
        <height>181</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;计算导论与程序设计课程设计 - 自动驾驶公交车调度系统（动画版）&lt;/p&gt;&lt;p&gt;组别：第四组&lt;/p&gt;&lt;p&gt;班级：2021211315&lt;/p&gt;&lt;p&gt;组长：梁晨锐 2021211321&lt;/p&gt;&lt;p&gt;组员：闫晨浩 2021211314&lt;/p&gt;&lt;p&gt;组员：孙懿 2021211315&lt;/p&gt;&lt;p&gt;2022年7月&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>191</width>
        <height>251</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">image: url(:/img/author1and2.png);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="play">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>220</y>
        <width>21</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="title_zh">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>511</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑 Light</family>
      <pointsize>26</pointsize>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#ffffff;&quot;&gt;自动驾驶公交车调度系统 动画版&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="title_en">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>70</y>
      <width>241</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Consolas</family>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#12c8fa;&quot;&gt;AutoBus Manager with UI&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>70</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Forte</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Version:1.0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="quit">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>20</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:img/quit.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>tabWidget</zorder>
   <zorder>logo</zorder>
   <zorder>title_zh</zorder>
   <zorder>title_en</zorder>
   <zorder>label</zorder>
   <zorder>quit</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
